build-support/bin/BUILD

```diff
@@ -14,13 +14,22 @@ python_sources(
     },
 )
 
+python_requirement(
+    name="chevron",
+    requirements=["chevron==0.14.0"],
+)
+
 python_tests(name="py_tests", overrides={"reversion_test.py": {"timeout": 90}})
 
 pex_binary(name="changelog", entry_point="changelog.py")
 pex_binary(name="check_banned_imports", entry_point="check_banned_imports.py")
 pex_binary(name="check_inits", entry_point="check_inits.py")
 pex_binary(name="deploy_to_s3", entry_point="deploy_to_s3.py")
-pex_binary(name="generate_all_lockfiles_helper", entry_point="_generate_all_lockfiles_helper.py")
+pex_binary(
+    name="generate_all_lockfiles_helper",
+    entry_point="_generate_all_lockfiles_helper.py",
+    dependencies=[":chevron"],
```

Comment:
Doesn't dep inference pick this up?

Reply:
It's not used at all by this binary. 

I think that Asher's thinking was "I want to make sure Chevron is included when I generate the lockfile, so...I'll add as a dep to this script." Really, this script only wraps `generate-lockfiles` goal, so adding the dep doesn't do anything.

Reverted.

Reply:
No... I got my pex targets confused
I meant to add that to the generate docs binary and this way avoid including it in the global requirement txt file